frontend/src/metabase/admin/settings/containers/SettingsEditorApp.jsx

```diff
@@ -185,8 +187,14 @@ export default class SettingsEditorApp extends Component {
         );
 
         // if this is the Updates section && there is a new version then lets add a little indicator
+        const shouldDisplayNewVersionIndicator =
+          slug === "updates" &&
+          newVersionAvailable &&
+          !MetabaseSettings.isHosted();
+
         let newVersionIndicator;
-        if (slug === "updates" && newVersionAvailable) {
+
+        if (shouldDisplayNewVersionIndicator) {
```

Comment:
optional nitpick:

Remove this `if` statement and change the jsx below

```
          {shouldDisplayNewVersionIndicator && (
            <span
              style={{ padding: "4px 8px 4px 8px" }}
              className="bg-brand rounded text-white text-bold h6"
            >
              1
            </span>
          )}
```

Reply:
Done.

Yeah, looking at this file as a whole, the impulse is to refactor a lot more.

Thanks for the good nudge!